dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/common/SqlQuery.java

```diff
@@ -58,15 +63,13 @@ public SqlQuery( final String statement, final Map<String, Object> params )
         log.debug( "PARAMS: " + params );
     }
 
-    /**
-     * @throws IllegalArgumentException if statement is null/empty/blank
-     */
     @Override
     public String statement()
     {
         return statement;
     }
 
+    @Override
     public Map<String, Object> params()
     {
         return params;
```

Comment:
can't we name these methods as regular getters ? In that case we will be able to remove and use `@Getter` annotation

Reply:
Done